.github/workflows/deploy-production.yml

```diff
@@ -19,7 +18,7 @@ jobs:
     # Map a step output to a job output
     outputs:
       api: ${{ steps.changes.outputs.api }}
```

Comment:
should this be `api-prod` & `api-sandbox`?

Reply:
This is just to identify the changed files, not related to environment.